components/TextFields/examples/MultilineTextFieldLegacyExample.m

```diff
@@ -197,6 +251,15 @@ - (void)viewDidLoad {
                     [[note.userInfo valueForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue];
                 self.scrollView.contentInset = UIEdgeInsetsMake(0, 0, CGRectGetHeight(frame), 0);
               }];
+  [[NSNotificationCenter defaultCenter]
+      addObserverForName:UIKeyboardWillChangeFrameNotification
+                  object:nil
+                   queue:[NSOperationQueue mainQueue]
+              usingBlock:^(NSNotification *_Nonnull note) {
```

Comment:
Nit: I would prefer to avoid the block syntax with NSNotificationCenter, does it make sense to use a selector instead?

Reply:
Since it's educational code, I wanted it to stay as simple as possible. Also, developers are moving to the block syntax for notifications so this will seem familiar to them.

Reply:
Ok. I had learned that the block API was more dangerous, so I've spent a bit of time migrating code away in the past.

Reply:
Done.